lib/config.ts

```diff
@@ -0,0 +1 @@
+export const MAX_SUPPORTED_FILE_SIZE: number = 2 * 1024 * 1024 * 1024;
```

Comment:
config.ts sounds like a configurable thing. This is not really configurable, since it's an implementation related limit.

Reply:
> config.ts sounds like a configurable thing. This is not really configurable, since it's an implementation related limit.

As I've answered here https://github.com/snyk/snyk-cpp-plugin/pull/48#discussion_r740026514 `common.ts` would make more sense